src/transport/SessionManager.h

```diff
@@ -171,6 +171,9 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     void ExpireAllPairingsForFabric(FabricIndex fabric);
     void ExpireAllPASEPairings();
 
+    // This API is used by UpdateNOC command, to invalid all sessions except the given one.
+    void ReleaseSessionForNodeExceptOne(const ScopedNodeId & node, const SessionHandle & stay);
```

Comment:
Given the description comment, shouldn't this be named `ReleaseSessionsForFabric` and the delayed "exception" can be an optional parameter `sessionWithDelayedRemoval`?